provider/cmd/pulumi-resource-aws/main.go

```diff
@@ -30,16 +29,15 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
 )
 
-//go:embed schema-embed.json
-var pulumiSchema []byte
-
-//go:embed schema-minimal-embed.json
-var pulumiMinimalSchema []byte
+//go:embed schema-*-embed.json
```

Comment:
The advantage of the `*` is that code still compiles even when minimal schema is not present on disk (such as upon checkout).